pages/api/v1/users/[id]/isMod.ts

```diff
@@ -0,0 +1,8 @@
+import type { NextApiRequest, NextApiResponse } from 'next'
+import Users from "../../../../models/users.ts"; /*i think this correctly requires the users model*/
+
+export default async (req: NextApiRequest, res: NextApiResponse) => {
+ const user = await Users.findOne({id: req.params.id});
```

Comment:
```suggestion
 const user = await Users.findOne({id: req.query.id});
```

Reply:
@MattA-Official uh isn't `[id]` folder resolve to req.params.id?
I'm sure it does. Probably you're forgetting how to use express and other servers...

Reply:
> @MattA-Official uh isn't `[id]` folder resolve to req.params.id?
> I'm sure it does. Probably you're forgetting how to use express and other servers...

With NextJS it is `query`, not `param`. This was pointed out by an automatic type check